File: learn-java-w3c/Strings.java
/**
 * Strings
 */
public class Strings {

    public static void main(String[] args) {
        String greeting = "Hello";
        String name = "John";

        // Using length
        System.out.println(greeting.length());
        // More methods
        System.out.println(greeting.toUpperCase());
        System.out.println(greeting.toLowerCase());
        System.out.println(greeting.indexOf("ll"));
        System.out.println(greeting + " " + name);
        System.out.println(greeting.concat(name));

    }
}

File: learn-C#-w3c/strings.cs
using System;

namespace MyApplication
{
    class Program
    {
        static void Main(string[] args)
        {
            string greeting = "Hello";
            Console.WriteLine(greeting.ToUpper());
            Console.WriteLine(greeting.ToLower());

            string firstname = "John";
            string lastname = "Wike";
            string name = firstname + lastname;
            Console.WriteLine(name);
            Console.WriteLine(string.Concat(firstname, lastname));
            Console.WriteLine($"My full name is: {firstname} {lastname}");




        }
    }
}